apps/api/src/services/InsightService.ts

```diff
@@ -8,9 +8,10 @@ import BaseService from './BaseService';
 
 export interface IInsightSearch {
     where: {
-        project: string | string[];
+        project?: string | string[];
+        end?: string;
     };
-    relations: string | string[];
+    relations?: string | string[];
```

Comment:
Should all of these be optional?

Reply:
Yes, if not you will be required to include all fields when searching.